doc/yjit/yjit.md

```diff
@@ -231,7 +245,7 @@ if each process only handles 1000 requests, `--yjit-call-threshold=1000` might b
 
 ### Decreasing --yjit-exec-mem-size
 
-`--yjit-exec-mem-size` specifies the JIT code size, but YJIT also uses memory for its metadata,
+The `--yjit-exec-mem-size` option specifies the JIT code size, but YJIT also uses memory for its metadata,
 which often consumes more memory than JIT code. Generally, YJIT adds memory overhead by roughly
 3-4x of `--yjit-exec-mem-size` in production as of Ruby 3.2. You should multiply that by the number
```

Comment:
@k0kubun would you say that the 3-4x figure is still accurate?

With the work we've done, it's maybe closer to 2x or 2-3x?

Reply:
It's closer than 3.2, yes, but it still seems to be in the range of 3-4x.

On SFR experimental clusters running very recent master, max yjit_alloc_size is 72MiB when max code_region_size is 28MiB. So yjit_alloc_size + code_region_size is 3.6x as large as code_region_size.